src/api/api.ts

```diff
@@ -755,6 +756,19 @@ export class OpenSeaAPI {
 
     const response = await req.send();
     if (!response.ok()) {
+      if (response.statusCode === 599) {
```

Comment:
I could also use the standard 429 status code here to make the clause future-proof.

Reply:
429 makes sense, not sure if it should be catching 599 for rate limit?

Reply:
This is the status code currently returned from the API (likely a proxy under your control sitting in front of the actual API).